client/src/helper/utils.js

```diff
@@ -54,21 +54,12 @@ export function checkIfDateIsDaysBeforeToday(date, daysBeforeToday) {
     (new Date().getTime() - new Date(date).getTime()) / 1000 / 60 / 60 / 24;
   return daysDifference < daysBeforeToday;
 }
-
-import sanitizeHtml from "sanitize-html";
 export function sanitizeComment(message) {
-  const sanitized = sanitizeHtml(message, {
-    allowedTags: ["a"],
-    allowedAttributes: {
-      a: ["href", "target"],
-    },
-  });
-
   // Create clickable links from link text
   const URLMatcher =
-    /^(?:(?:https?):\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,}))\.?)(?::\d{2,5})?(?:[/?#]\S*)?$/i;
+    /(?:(?:https):\/\/)(?:\S+(?::\S*)?@)?(?:(?!(?:10|127)(?:\.\d{1,3}){3})(?!(?:169\.254|192\.168)(?:\.\d{1,3}){2})(?!172\.(?:1[6-9]|2\d|3[0-1])(?:\.\d{1,3}){2})(?:[1-9]\d?|1\d\d|2[01]\d|22[0-3])(?:\.(?:1?\d{1,2}|2[0-4]\d|25[0-5])){2}(?:\.(?:[1-9]\d?|1\d\d|2[0-4]\d|25[0-4]))|(?:(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)(?:\.(?:[a-z\u00a1-\uffff0-9]-*)*[a-z\u00a1-\uffff0-9]+)*(?:\.(?:[a-z\u00a1-\uffff]{2,}))\.?)(?::\d{2,5})?(?:[/?#]\S*)?/i;
```

Comment:
Ah... localhost fehlt hier in diesem Monster^^ (erbraucht min. den Punkt zwischen Namen und TLD) 
Deshalb geht der Link (s.o.) auch nicht.

Reply:
genau, aber localhost brauchen wir ja auch nicht verlinken;)

Reply:
Zu Testzwecken wäre es ganz nett

Reply:
Aber warum was testen, was nachher nicht in Produktion soll? Nimm doch einfach irgendeinen link. ob du netz hast oder nicht, ist ja wurst, geht ja nur darum dass er klickbar ist.